### PR TITLE
Frankreed/Allow coach to view all assignments (with unify all alerts merged)

### PR DESCRIPTION
**Pull request by FrankreedX**
EDIT: See https://github.com/Golf-Drill-Challenge-App/Golf-App/pull/239

- Keeping this PR open just for reference, but probably not merging it unless the "merge all" PR above has issues?

### CODE REVIEW

**Comment by Gehrkej** [comment]
Not sure why this is looking to merge into the all time record branch and not layout

**Comment by FrankreedX** [comment]
because it's rebased off of all time record

**Comment by FrankreedX** [comment]
if all time record merges first then this branch's target will change

File: components/assignmentList.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        invalidateKeys={invalidateKeys}
+        text={"No drills assigned"}
+      />
+    );
+  }
+
+  const stackedPfp = (playerList) => {
+    const numPfp = Math.min(3, playerList.length);
+
+    const pfpArr = playerList.slice(0, numPfp).map((player, index) => {
```
Comment (hannacol) [inline components/assignmentList.js:97]: Are we able to make it display 3 image cards and then the last card displays "+{numAssignedPlayers - 3}" if there are more than 3 players assigned?

**Review by hannacol — CHANGES_REQUESTED** [review]
Looks good overall. Couple things I would change styling on for better understanding for a user

**Review by solderq35 — COMMENTED** [review]
image

Just glancing at some of these commits, are some of these duplicated from layout branch (since all time records was already merged into layout)?

If we're short on time though, I guess just confirm everything works and forget about git history clutter.

**Pull request closed without merging**